apps/enc.c

```diff
@@ -99,6 +101,8 @@ int enc_main(int argc, char **argv)
     int ret = 1, inl, nopad = 0;
     unsigned char key[EVP_MAX_KEY_LENGTH], iv[EVP_MAX_IV_LENGTH];
     unsigned char *buff = NULL, salt[PKCS5_SALT_LEN];
+    int pbkdf2 = 0;
+    int iter = 1;
```

Comment:
Assign zero.  The reason for this is that the user might enter something like `-iter 1 -pbkdf2` and be very dissapointed with the result (because your code changed it to 1000).